Core/src/org/sleuthkit/autopsy/datamodel/InterestingHits.java

```diff
@@ -324,14 +302,13 @@ public SetNameNode(String setName) {//, Set<Long> children) {
         }
 
         private void updateDisplayName() {
-            int sizeOfSet = interestingResults.getArtifactIds(setName, BlackboardArtifact.Type.TSK_INTERESTING_ARTIFACT_HIT.getDisplayName()).size()
-                    + interestingResults.getArtifactIds(setName, BlackboardArtifact.Type.TSK_INTERESTING_FILE_HIT.getDisplayName()).size();
+            int sizeOfSet = interestingResults.getArtifactIds(setName).size();
             super.setDisplayName(setName + " (" + sizeOfSet + ")");
         }
 
         @Override
         public boolean isLeafTypeNode() {
-            return false;
+            return true;
```

Comment:
A SetNameNode would not be a leaf node because it has analysis result node children, right? It's interesting that it doesn't seem to matter that this has been set to false...it seems like this would break IsLeafItemVisitor...

Reply:
Never mind, there is a documentation problem here. I will address it later.